canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -265,6 +280,20 @@ class DropDown extends React.Component {
 					disabled={this.props.state === STATES.DISABLED || this.disableEmptyListDropdown}
 					type="default"
 					items={dropDown.options}
+					itemToString={(item) => (item ? (
```

Comment:
Why is `itemToString` and `itemToElement` both needed and both the same code?  I would think `itemToString` would be a string?